ci-operator/jobs/openshift/odo/openshift-odo-master-presubmits.yaml

```diff
@@ -14,6 +14,7 @@ presubmits:
       pj-rehearse.openshift.io/can-be-rehearsed: "true"
     name: pull-ci-openshift-odo-master-unit
     rerun_command: /test unit
+    run_if_changed: ^(?:[^d]+|d(?:$|[^o]|o(?:$|[^c]|c(?:$|[^s]))))*$
```

Comment:
What about README.adoc. Looks good to me otherwise.

Reply:
should we consider a regex that discard any string that contains "doc" because "adoc" will be too specific and may run for the changes of markdown files
_cc: @mohammedzee1000

Reply:
Makes sense, though you should probably consider https://github.com/openshift/odo/tree/master/cmd/cli-doc. Not sure if this path is ever tested or not, but as it is code, it should probably not be skipped

Reply:
I am afraid it will be skipped,

Reply:
Ok. I went through and it is not tested any case. So lets let it be for now. Please make sure to record this in an issue

Reply:
@anandrkskd we care the most about the `*.adoc` and `*.md` files under https://github.com/openshift/odo/tree/master/docs/. A PR modifying/adding documentation under any of the directories under this directory should not be going through CI.

Reply:
yeah this PR will not run CI if any files from `odo/docs` is changed.